index.html

```diff
@@ -1,305 +1,295 @@
+                        </li>
+                        <li>
+                            <a href="#"><img src="./img/hn.svg" alt="HN"></a>
+                        </li>
                     </ul>
-                  </form>
                 </div>
-              </div>
-            </section>
-          </section>
-        </body>
-      </html>
\ No newline at end of file
+
+                <a href="#" class="btn">Get my resume</a>
+            </div>
+
+            <div class="languages">
+                <div>
+                    <div class="heading">
+                        <span>Languages</span>
+                        <img src="./img/Union_d.png" class="chevron-down" alt="chevron down" />
+                    </div>
+
+                    <div class="individual-languages">
+                        <div>
+                            <img src="./img/Java.png" alt="Java">
+                            <p>JavaScript</p>
+                        </div>
+
+                        <div>
+                            <img src="./img/html.png" alt="HTML">
+                            <p>HTML</p>
+                        </div>
+
+                        <div>
+                            <img src="./img/css.png" alt="CSS">
+                            <p>CSS</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="grey-bottom-border">
+                    <div class="heading">
+                        <span>Frameworks</span>
+                        <img src="./img/Union.png" class="chevron-right" alt="chevron right" />
+                    </div>
+                </div>
+                <div class="grey-bottom-border">
+                    <div class="heading">
+                        <span>Skills</span>
+                        <img src="./img/Union.png" class="chevron-right" alt="chevron right" />
+                    </div>
+                </div>
+            </div>
+        </section>
+        <footer class="contact-me">
+
+            <div class="contact-text">
+                <h2>Contact me</h2>
+
+                <p>If you have an application you are interested in developing, a feature that you need built
+                    or a project that you need coding. I'd love to help with it.</p>
+            </div>
+
+            <div class="form">
+                <form action="https://formspree.io/f/xoqrdelo" method="post">
+                    <ul>
+                        <li><input type="text" name="name" placeholder="Your Name" maxlength="30" required></li>
+                        <li><input type="email" name="email" placeholder="Your Email" required></li>
+                        <li><textarea name="message" placeholder="Write your message here" maxlength="500"
+                                required></textarea></li>
+                        <li><button class="button-submit" type="submit">Get It Touch</button></li>
+                    </ul>
+                </form>
```

Comment:
- [ ] Please add some width to the form and some few 🤌  `padding` to separate the title from the `p` and `p` form the form, they are so close and that is not fine for the user readability

- [ ] YOUR DESIGN  
<img width="533" alt="contsc" src="https://user-images.githubusercontent.com/79058364/155616625-48585a12-256c-4a6a-9815-971c5e0afa1c.png">

- [ ] THE FIGMA
<img width="475" alt="GIF" src="https://user-images.githubusercontent.com/79058364/155617385-6cf3e112-0c0f-47a0-8e54-9d7953c9eca6.png">

Reply:
@BenMukebo . Please review changes that I have made and additional commits